docs.bruno/test-bruno.bru

```diff
@@ -0,0 +1,11 @@
+meta {
+  name: test-bruno
+  type: http
+  seq: 1
+}
+
+get {
+  url: http://localhost:3000/ping
```

Comment:
It might be a good idea to replace the hardcoded URL with a variable or a constant. This would make the code more flexible and easier to maintain. [important]